torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3427,6 +3496,16 @@ def reference_lgamma(x):
                # sort does not correctly warn when resizing out= inputs
                SkipInfo('TestCommon', 'test_out'),
            )),
+    OpInfo('put',
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           supports_out=False,
+           check_batched_gradgrad=False,  # vmap complains of the sizes
+           sample_inputs_func=sample_inputs_put),
+    OpInfo('take',
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           supports_inplace_autograd=False,
```

Comment:
This can probably be removed. A recent update now checks that the op has an inplace variant BEFORE checking whether it supports inplace grad or not, so ops without inplace variants no longer need to set this.